libmscore/layoutbreak.cpp

```diff
@@ -65,7 +68,7 @@ void LayoutBreak::write(XmlWriter& xml) const
       xml.stag(this);
       Element::writeProperties(xml);
 
-      for (auto id : { Pid::LAYOUT_BREAK, Pid::PAUSE, Pid::START_WITH_LONG_NAMES, Pid::START_WITH_MEASURE_ONE })
+      for (auto id : { Pid::LAYOUT_BREAK, Pid::PAUSE, Pid::START_WITH_LONG_NAMES, Pid::START_WITH_MEASURE_ONE, Pid::FIRST_SYSTEM_INDENTATION})
```

Comment:
I'd bet the 'missing' space at the end of the list causes a code style issue in master...

Reply:
My head is fully at <code>master</code> yet :-), good catch.